.gitignore

```diff
@@ -14,3 +14,5 @@ npm-debug.log
 *.log
 
 /.nyc_output/
+node_modules/
```

Comment:
There's a `/node_modules` entry higher up this file, could you edit that one / combine with this line?

Reply:
I removed mine and edited the one above. Sharp eye. Thanks